app/pb_kits/playbook/pb_text_input/_text_input.html.erb

```diff
@@ -1,5 +1,8 @@
 <%= content_tag(:div,
-    class: object.classname) do %>
+    aria: object.aria,
+    class: object.classname,
+    data: object.data,
+    id: object.id) do %>
```

Comment:
I'm not sure about the id change. Going to check with some folks but I believe the ID will be needed for the input in many cases, not the wrapping div. An example is select: https://github.com/powerhome/playbook/blob/v4.15.0/app/pb_kits/playbook/pb_select/_select.html.erb#L22